spec/requests/proceeding_types/searches_controller_swagger_spec.rb

```diff
@@ -0,0 +1,223 @@
+require "swagger_helper"
+
+RSpec.describe "ProceedingTypes::SearchesController", type: :request do
```

Comment:
For consistency with other renaming of top level describe block would this be more appropriate.

```suggestion
RSpec.describe "searches", type: :request do
OR
RSpec.describe "proceeding_types/searches", type: :request do
```